### DISCUSSION TITLE
Welcome to Cookie-Logger Discussions!

### DISCUSSION THREAD

**Discussion by p-qq** [p-qq/Cookie-Logger#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by notanaltac2** [comment]
does this help
__

**Reply by p-qq** [reply]
yes i see the question

**Comment by notanaltac2** [comment]
or this one

**Reply by p-qq** [reply]
it's working
